src/rf/apps/core/models.py

```diff
@@ -133,6 +133,16 @@ class Meta:
     thumb_large_key = CharField(max_length=255, blank=True, default='',
                                 help_text='S3 key for large thumbnail')
 
+    def retry_possible(self):
+        """
+        Returns true if the layer failed during the EMR steps.
+        """
+        thumbnailed_images = self.layer_images.filter(
+            status=enums.STATUS_THUMBNAILED)
+        all_thumbnailed = self.layer_images.count() == \
+            thumbnailed_images.count()
+        return self.status == enums.STATUS_FAILED and all_thumbnailed
```

Comment:
If we are tweaking validation logic on the backend, then retry would be very helpful so that we don't have to reupload files.

I also don't think we should be placing any significance on thumbnails. They have no impact on whether layers get tiled or not.